deploy/charts/cert-manager/values.schema.json

```diff
@@ -648,6 +651,11 @@
       "description": "enableServiceLinks indicates whether information about services should be injected into the pod's environment variables, matching the syntax of Docker links.",
       "type": "boolean"
     },
+    "helm-values.enabled": {
+      "default": true,
+      "description": "Field that can be used as a condition when cert-manager is a dependency. This definition is only here as a placeholder such that it is included in the json schema. See https://helm.sh/docs/chart_best_practices/dependencies/#conditions-and-tags for more info.",
+      "type": "boolean"
+    },
```

Comment:
The opentelemetry implementation defaults to false....but what does "default" mean in this context anyway?
 * https://github.com/open-telemetry/opentelemetry-helm-charts/pull/262/files#diff-278a127a2cf2ecde9379510e3ea99f99312940bcf24b4463fff9222d5150c9c4R7-R10

Reply:
As I now, Helm don't care about default in schema. It's only for doc purpose.

https://github.com/helm/helm/issues/8218